README.md

```diff
@@ -1,42 +1,36 @@
 # Isilon Data Insights Connector
-The isi_data_insights_d.py script controls a daemon process that can be used to query multiple OneFS clusters for statistics data via the Isilon's Platform API (PAPI). It then has a pluggable module for processing the results of those queries. An example stat processor that sends the query results to an instance of InfluxDB is provided along with an example Grafana dashboard.
+The isi_data_insights_d.py script controls a daemon process that can be used to query multiple OneFS clusters for statistics data via the Isilon's Platform API (PAPI). It then has a pluggable module for processing the results of those queries. The provided stat processor, defined in influxdb_plugin.py, sends query results to an InfluxDB backend. Additionally, several Grafana dashboards are provided to make it easy to monitor the health and status of your Isilon clusters.
 
 # Install Dependencies Locally
 * sudo pip install -r requirements.txt
-* Install Isilon SDK Python language bindings for OneFS 8.0.X and/or 7.2.X (see SDK version help below).
-* Instructions for SDK installation: https://github.com/Isilon/isilon_sdk
 
 # Install Virtual Environment
```

Comment:
Recommend addressing Python 2/3 compatibility. The stats connector is not python 3 ready. If it's not too much work, recommend making Python 3 compatible at some point. Most of the work would probably be surrounding print statements with parenthesis. One reason is that if I create a virtualenv on a box with python 3 installed, python 3 will be used in the virtualenv. Supposedly Python 3 will start becoming the default install on Linux distros in the future.